profiles/bridge.nix

```diff
@@ -0,0 +1,39 @@
+{ config, pkgs, inputs, ... }:
+let
+  profiles = "/nix/var/nix/profiles/per-user/deploy";
```

Comment:
Hm, there is no `deploy` user configured. Is by added in the app module?

Reply:
Now they are configured in all profiles.